working-groups/kafka/CHARTER.md

```diff
@@ -0,0 +1,39 @@
+# Kafka Eventing Working Group
+
+# Motivation/Abstract
+
+Within the knative-sandbox org, Kafka based eventing currently includes 2 channel implementations, 2 sources implementations, 1 broker and 1 sink with more than three vendors and multiple users for production workloads. With that, it is desired to have a dedicated forum for Kafka based Eventing which can be really beneficial both for exposure and focus.
+
+This is a proposal to create a new working group that would be a sub working group of Eventing and would report a brief status summary to the Eventing WG being a sub working group.
```

Comment:
If merged, it's not a proposal anymore. You should describe the mission as a given here or just skip this sentence as there is a mission statement already below.